src/pages/experiments/[experimentId]/data-exploration/components/header/index.jsx

```diff
@@ -100,6 +100,7 @@ const Header = (props) => {
           <Space>
             <FeedbackButton />
             <Dropdown
+              trigger={['click']}
               key='search-menu-dropdown'
               overlay={searchMenu}
               visible={addMenuVisible}
```

Comment:
I think the problem is the visible stuff here. Did you try to remove it entirely? Because I think that removing it entirely might actually be the fix we need.